src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+    >
+      <label for="big__search__input">
+        <img
+          src="images/Search.svg"
+          alt="Search"
+          class="big__icon"
+        >
+        <input
+          class="large__input search__input"
+          data-qa="keypress"
+          type="text"
+          id="big__search__input"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="small"
+    >
+      <label for="small__search__input">
+        <img
+          src="images/Search.svg"
+          alt="Search"
+          class="small__icon">
```

Comment:
```suggestion
          class="small__icon"
        >
```
![image](https://user-images.githubusercontent.com/99652389/178441377-1e6fa5eb-200a-445c-954f-61f412774551.png)
`checklist.rm`